### DISCUSSION TITLE
How to set exact values list on the y-axis

### DISCUSSION THREAD

**Discussion by vciioffi** [patrykandpatrick/vico#872] [Questions]
### Question

Having a list of values how can I set this exact list as the values displayed on the y-axis?
With my current implementation there is always the same gape between each value so the graphic always looks the same:
Version: 2.0.0-alpha.19

`    fun setUpChart(player: PlayerVO): CustomChartDialog {
        val modelProducer = CartesianChartModelProducer.build()
        val axisValueOverrider = AxisValueOverrider.fixed(minY = player.pricingHistory.min().toFloat(), maxY = player.pricingHistory.max().toFloat())
        val axisItemPlacer = AxisItemPlacer.Vertical.count()
        with(binding.chartView){
            (chart?.layers?.get(0) as LineCartesianLayer?)?.axisValueOverrider = axisValueOverrider
            (chart?.startAxis as VerticalAxis).itemPlacer = axisItemPlacer
            runInitialAnimation = false
            this.modelProducer = modelProducer

        }
        modelProducer.tryRunTransaction { lineSeries { series(player.pricingHistory.reversed()) } }

        modelProducer.tryRunTransaction {
            lineSeries {
                series(x = listOf(1, 2, 3, 4), y = player.pricingHistory.reversed())
            }
        }
        return this
    }`

### Vico version(s)

2.0.0-alpha.19

### UI framework(s)

Views

**Comment by Gowsky** [comment]
Hi @vciioffi. Please see `AxisItemPlacer.Vertical`.
